integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -326,6 +326,7 @@ func BuildAddDONArgs(
 			schedule,
 			oracles,
 			encodedOffchainConfig,
+			nil,
```

Comment:
```suggestion
			nil, // maxDurationInitialization
```

Reply:
Just a small nit, especially for these functions with 10+ arguments.

Reply:
Updated, but FWIW goland does this:
![image](https://github.com/user-attachments/assets/1e03a15c-0ad5-4afe-90b7-a6054efb1927)